index.html

```diff
@@ -178,121 +180,125 @@ <h3 class="card-title">Multi-Post Stories</h3>
         </article>
       </div>
     </section>
-
-    <section class="about">
-      <div class="container-about">
-        <div class="me">
-          <h1 class="title">About Myself</h1>
-          <p class="about-description">
-            Hello I’m a software developer! I can help you build a product ,
-            feature or website Look through some of my work and experience! If
-            you like what you see and have a project you need coded, don’t
-            hesitate to contact me.
-          </p>
-          <h2 class="connect">Let's connect</h2>
-          <ul class="social">
-            <li>
-              <a href="twitter.com"
-                ><img src="./images/svg/twitter.svg" alt="twitter"
-              /></a>
-            </li>
-            <li>
-              <a href="linkedin.com"
-                ><img src="./images/svg/Linkedin.svg" alt="LinkedIn"
-              /></a>
-            </li>
-            <li>
-              <a href="https://medium.com"
-                ><img src="./images/svg/medium.svg" alt="Medium"
-              /></a>
-            </li>
-            <li>
-              <a href="github.com"
-                ><img src="./images/svg/github.svg" alt="Github"
-              /></a>
-            </li>
-            <li>
-              <a href="angellist.com"
-                ><img src="./images/svg/angellist.svg" alt="Angel List"
-              /></a>
-            </li>
-          </ul>
-          <div class="about-actions">
-            <a href="#" class="primary-button">Get my resume</a>
-          </div>
-        </div>
-        <div class="skills">
-          <div class="about-title-container">
-            <h3 class="about-skills">Languages</h3>
-            <img src="./images/svg/arrow.svg" />
-          </div>
-          <ul class="about-languages">
-            <li class="about-item">
-              <img src="./images/languages/JS.png" alt="Javascrip" />JavaScript
-            </li>
-            <li class="about-item">
-              <img src="./images/languages/HTML.png" alt="HTML" />HTML
-            </li>
-            <li class="about-item">
-              <img src="./images/languages/CSS.png" alt="CSS" />CSS
-            </li>
-          </ul>
-          <div class="about-title-container border-b">
-            <h3 class="about-skills">Frameworks</h3>
-            <img src="./images/svg/arrow-right.svg" />
+    <div class="blank">
+      <section class="about">
+        <div class="container-about">
```

Comment:
- [ ] I will implore you to make the content width to be 40 and 60 percent just as it is for the design template.

### Your work
![image](https://github.com/DHE19/portfolio-microverse/assets/86200356/6e496214-9f93-46b7-b7d6-d2806767f647)


### Figma
![image](https://github.com/DHE19/portfolio-microverse/assets/86200356/46fc76e5-2f69-4f4b-90e6-821a95648cb4)